RecoLocalFastTime/FTLCommonAlgos/src/MTDTimeCalib.cc

```diff
@@ -0,0 +1,86 @@
+#include "RecoLocalFastTime/FTLCommonAlgos/interface/MTDTimeCalib.h"
+
+#include "DataFormats/ForwardDetId/interface/BTLDetId.h"
+#include "DataFormats/ForwardDetId/interface/ETLDetId.h"
+
+#include "Geometry/CommonDetUnit/interface/GeomDetType.h"
+#include "Geometry/MTDGeometryBuilder/interface/ProxyMTDTopology.h"
+#include "Geometry/MTDGeometryBuilder/interface/RectangularMTDTopology.h"
+#include "Geometry/MTDNumberingBuilder/interface/GeometricTimingDet.h"
+
+MTDTimeCalib::MTDTimeCalib(edm::ParameterSet const& conf, 
+			   const MTDGeometry* geom, const MTDTopology* topo):
+  geom_(geom),
+  topo_(topo),
+  btlTimeOffset_( conf.getParameter<double>("BTLTimeOffset") ),
```

Comment:
It could be a good idea to add a comment to the description [1] with at least the measurement units to be considered for these parameters

[1]
https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideConfigurationValidationAndHelp#Adding_Comments

Reply:
ID 62820d3f TIME_OFFSET -0.215375
ID 63042980 TIME_OFFSET 0.0066
ID 63053000 TIME_OFFSET 0.0066
ID 63053080 TIME_OFFSET 0.0066
ID 63440580 TIME_OFFSET 0.0066
ID 629c2a3f TIME_OFFSET -0.215375
ID 62d50e15 TIME_OFFSET -0.215375

Time offsets appear applied as expected for BTL & ETL

Reply:
> Do you understand why?

ETL anyway looks changed in my opinion as expected (shifts to the right, the fact that some bins do not move much I think is connected to the binning (the shift is only 6ps so small compared to the bin size))
![all_oldvsnew_ttbar14tev2023d35wf27434p0c_ftlrechitssorted_mtdrechits_ftlendcap_reco_obj_obj_time-2](https://user-images.githubusercontent.com/4571680/51267002-ff57c900-19bc-11e9-866a-1e559096d9b4.png)